README.rst

```diff
@@ -63,20 +70,21 @@ Open edX compatibility notes
 
 
 Usage
```

Comment:
I think we should improve the structure of this section to explain the two main ways it can be used, something like this:

```suggestion
Usage
=====

**eox-audit-model** can be used to audit any execution of a method or function. This will create a database record with the following information:
- **Status**: Indicates whether the process was successful or not.
- **Action**: The given string to identify the process.
- **Timestamp**: The date and time the action was executed.
- **Method Name**: The name of the method or function that was executed.
- **Captured Log**: The logs generated during the execution.
- **Traceback Log**: Contains the traceback information if an exception occurred.
- **Site**: The current site where the action was performed.
- **Performer**: The user who initiated the method, based on `request.user`.
- **Input**: The values used to execute the method.
- **Output**: The value returned by the method.
- **IP**: The IP address from which the action was initiated.

There are two primary ways to use **eox-audit-model**:

#### 1. Direct Method Call

You can log an action directly by importing the model and calling the `execute_action` method. This method requires several parameters to log the information:

- `action`: A string describing the action, e.g., `'Add view info'`.
- `method`: The method being executed.
- `parameters`: A dictionary containing positional arguments (`args`) and keyword arguments (`kwargs`).
- `notes`: An optional list of dictionaries for storing custom information.

Example:
...

#### 2. Using the Decorator

**eox-audit-model** also provides a decorator that can be used to automatically log method executions. The decorator handles calling the `execute_action` method behind the scenes and saves the information for you.

Example:
....

```